src/Fonts/ShadowStyleFonts.ts

```diff
@@ -0,0 +1,15 @@
+import { css } from "styled-components";
+import DefaultFont from "./DefaultFont";
+
+// CSS Attributes which normally being inherited from html/body should
+// aren't used in ShadowDom. It's encapsulation prevents a natural inheritance.
+// All attributes which are also global in Web Components should be
+// added to the Shadow Styles
+const ShadowStylesFonts = css`
+  input,
+  textarea {
```

Comment:
does this affect input and textarea only?

Reply:
Nice that you adressing this ... yes indeed.

It is a straight convert from the original file but it doesn't make sense for me either. It is the default font, which is also used in `body` ... therefore I would suggest to get rid of `input, textarea` and introduce him as default standard for everything.